Source/OCMock/OCClassMockObject.h

```diff
@@ -21,6 +21,7 @@
     Class  mockedClass;
     Class  originalMetaClass;
     Class  classCreatedForNewMetaClass;
+    BOOL   stoppedMocking;
```

Comment:
It seems unnecessary to remember this in a specific field. If `stopMocking` was ever invoked the `invocations` instance variable is nil. I'll make that change when merging.

Reply:
In fact, there is a method for this purpose already: `assertInvocationsArrayIsPresent`.